### ISSUE TITLE
Time for timeout parameters should be in ms in command line help

### ISSUE THREAD

**Issue by oflorko** [SeleniumHQ/selenium#4891]
## Meta -
OS:  Any console

Selenium Version:  >= 3.0.0

Browser:  N/A

Browser Version:  N/A

## Expected Behavior -
In console help all timing parameters should be in ms.
## Actual Behavior -
Wrong time dimensions for parameters related to timeouts in console help. 
 -browserTimeout
       <Integer> in seconds
  -timeout, -sessionTimeout
       <Integer> in seconds
## Steps to reproduce -

Type in command line `java -jar selenium-server-standalone-3.6.0.jar --help`

**Comment by barancev** [comment]
Why should they be in milliseconds?

**Comment by mmerrell** [comment]
If I’m correct, Svetla is indicating that they’re documented incorrectly. She attended the Grid workshop in Berlin and reported this issue during the workshop, but I didn’t have time to investigate then. 

I know that some options are in ms and some are in seconds, and I once went through and reconciled the code with the options, but she indicated that some were incorrectly documented. I thought it would be worth double-checking on the code side. I can take this one. 
MM

**Comment by kool79** [comment]
In grid v2.x all timeouts in json configuration files were in ms. But during migration to grid v3.8 I found that `timeout` () and `browserTimeout` now should be in seconds (in json).  I don't expect those changes because all parameter's names remain the same. I found this only after exploring source code of grid server. (I dont trust docs because they have erorrs before). It is very uncomfortable and error-prone to have different time units in the same configuration file: some timeouts in ms (nodeStatusCheckTimeout, unregisterIfStillDownAfter, newSessionWaitTimeout etc) while others (timeout, browserTimeout) use seconds.
  For example, official appium documentation (at the moment, http://appium.io/slate/en/master/?ruby#selenium-grid) contains `timeout` in seconds 
 I think this happens when code for configuration/cli parsing was refactored.  Because both those  parameters  were  `ms` in json but `sec` in command line interface.
  So for now we have a lot of people//documentation/guides with old configs (with `ms`), while some peoples switched to `sec`. New config is very confusing (see for  example , #2973)
